consensus-types/blocks/execution.go

```diff
@@ -824,26 +826,33 @@ func PayloadToHeaderElectra(payload interfaces.ExecutionDataElectra) (*enginev1.
 		return nil, err
 	}
 
+	consolidationRequests := payload.ConsolidationRequests()
+	consolidationRequestsRoot, err := ssz.ConsolidationRequestsSliceRoot(consolidationRequests, fieldparams.MaxConsolidationRequestsPerPayload)
```

Comment:
MaxConsolidationRequestsPerPayload sounds like it should be network specific ( mainnent vs minimal) shouldn't we have that there? if not fieldparams seems like a constant if it's always the same, why do we need to pass it in as an argument?

Reply:
nvm i see above it's the same